### ISSUE TITLE
Re-running library from `--reanalyse` to recreate match between run

### ISSUE THREAD

**Issue by hguturu** [vdemichev/DiaNN#754]
When DIA-NN is run with `--reanalyse` it outputs a library that is used for the second pass.

If I take this library and search the same files, I get different output. Is there a way to recreate the second pass without having to re-run the whole library?

**Comment by vdemichev** [comment]
No, but the output you get this way is comparable. Please use the main report & filtering advised for MBR.

**Comment by hguturu** [comment]
That is good to know its comparable. Is there a summary of what drives the difference? I noticed the number of decoys are different between the two ways of running it. Is that the difference with different convergence of the target/decoy model or are there other differences like stricter search parameters?
